lib/src/layer/tile_layer/tile_layer.dart

```diff
@@ -740,3 +746,9 @@ class _TileLayerState extends State<TileLayer> with TickerProviderStateMixin {
   bool _outsideZoomLimits(num zoom) =>
       zoom < widget.minZoom || zoom > widget.maxZoom;
 }
+
+double _distanceSq(TileCoordinates coord, Point<double> center) {
+  final dx = center.x - coord.x;
+  final dy = center.y - coord.y;
+  return dx * dx + dy * dy;
+}
```

Comment:
i think a well described simple pure function does not need a comment explaining what it does. that is just excessive. however, if we already have a distance function i think we should avoid making a new one if possible!

Reply:
This is certainly not the most cirtical optimization but it's very common in GIS application to work with squared distances rather than distances (look at any rtree). The sqrt doesn't change the order.

Reply:
Even through I wouldn't completely agree, I see your point. 
Therefore I'm fine with not having these private functions documented with a comment.